media/css/mozorg/home/home-a.less

```diff
@@ -0,0 +1,1097 @@
+#nav-main-menu {
+    background: #fff;
+    box-shadow: 2px 2px 0 rgba(0, 0, 0, .2);
+    left: -300px;
+    overflow: visible;
+    position: absolute;
+    top: 60px;
+    width: 200px;
+    z-index: 99;
+
+    li {
+        border-bottom: 1px solid #ccc;
+
+        &:last-child {
+            border: 0;
+        }
+    }
+
+    a {
+        background: #fff;
+        color: @color-text-primary;
+        display: block;
+        padding: 10px 20px;
+        text-decoration: none;
+        -webkit-transition: background-color 150ms ease, border-width 150ms ease, color 150ms ease;
+                transition: background-color 150ms ease, border-width 150ms ease, color 150ms ease;
+
+        &:hover,
+        &:focus {
+            background-color: #f5f5f5;
+            border-right: 6px solid @color-mozred;
+            color: @color-mozred;
+            text-decoration: underline;
+        }
+    }
+}
+
+.masthead-nav-main:hover #nav-main-menu,
+.masthead-nav-main .toggle:focus + #nav-main-menu {
+    left: auto;
+}
+
+.js #nav-main-menu {
+    left: auto;
+    display: none;
+}
+
+@media @mq-tablet {
+    .masthead-nav-main {
+        position: static;
+
+        .toggle {
+            display: none;
+        }
+    }
+
+    #nav-main-menu,
+    .js #nav-main-menu {
+        .clearfix;
+        .font-size(14px);
+        background: transparent;
+        box-shadow: none;
+        display: block;
+        margin: 0 20px;
+        padding: 25px 0 15px;
+        position: static;
+        width: auto;
+
+        li {
+            .border-box;
+            border: 0;
+            float: left;
+            padding: 0 5px;
+            text-align: center;
+            width: 33.33%;
+        }
+
+        a,
+        a:hover,
+        a:focus {
+            background: transparent;
+            border: 0;
+            color: #fff;
+            padding: 0;
+        }
+    }
+}
+
+
+// Page nav
+.masthead-nav-page {
+    .fira-sans-light;
+    background: @color-mozred;
+    margin: 0 -20px;
+    text-align: center;
+    line-height: 1.1;
+
+    a {
+        .border-box;
+        display: block;
+        height: 100%;
+        padding: 10px;
+        margin: 0;
+        text-decoration: none;
+        vertical-align: middle;
+        -webkit-transition: background-color 150ms ease, border-width 150ms ease, color 150ms ease;
+                transition: background-color 150ms ease, border-width 150ms ease, color 150ms ease;
+
+        &:hover,
+        &:focus {
+            text-decoration: underline;
+        }
+    }
+
+    .primary {
+        .flexbox;
+
+        li {
+            float: left;
+            padding: 1px;
+            width: 33.33%;
+        }
+
+        a {
+            background-color: @color-mozred-bright;
+
+            &:hover,
+            &:focus {
+                background-color: @color-mozred;
+            }
+        }
+    }
+
+    .secondary {
+        .flexbox;
+        margin: 0;
+        text-transform: uppercase;
```

Comment:
Any reason why these are uppercase? During the recent download button overhaul to make things consistent across mozorg, using uppercase was ruled a big no-go (particularly in relation to download buttons). Maybe this is a UX question for Holly?